features/checkout/paying_for_order/preventing_not_available_payment_method_selection.feature

```diff
@@ -8,6 +8,7 @@ Feature: Preventing not available payment method selection
         Given the store operates on a single channel in "United States"
         And the store has a product "PHP T-Shirt" priced at "$19.99"
         And the store allows paying with "Paypal Express Checkout"
+        And the store allows paying with "Bank transfer"
```

Comment:
Why do we need to add this step?

Reply:
Because if we don't have any available payment method we don't create a payment for the order, without it I had a problem with checout steps